solver/src/settlement_simulation.rs

```diff
@@ -85,18 +90,27 @@ pub async fn simulate_settlements(
         .into_iter()
         .map(|(future, transaction_builder)| {
             future.now_or_never().unwrap().map(|_| ()).map_err(|err| {
-                let err = Error::new(err);
-                match block {
-                    Block::FixedWithTenderly(block) => {
-                        err.context(tenderly_link(block, network_id, transaction_builder))
-                    }
-                    Block::LatestWithoutTenderly => err,
-                }
+                Error::new(err).context(tenderly_link(block, network_id, transaction_builder))
             })
         })
         .collect())
 }
 
+fn method(
```

Comment:
This function name seems quite unspecific, maybe get_settle_method() would be a better name?